internal/cli/internal/command/open/open.go

```diff
@@ -0,0 +1,67 @@
+package open
```

Comment:
This `open.go` file essentially duplicates the contents of `apps/move.go`. There's no reason to duplicate this, please do check what we've done for [`move.go`](https://github.com/superfly/flyctl/blob/c98d6cadd9d491abfcf7e8f3060d79dee19ad82b/internal/cli/internal/command/move/move.go). In your case, both the code for the constructor as well as the runner are the same so you can minimize the duplicated code even further.